x-pack/plugin/core/template-resources/src/main/resources/watch-history-no-ilm.json

```diff
@@ -1,6 +1,6 @@
 {
   "index_patterns": [ ".watcher-history-${xpack.watcher.template.version}*" ],
-  "priority": 2147483646,
+  "priority": 2147483647,
```

Comment:
We should probably leave this one out for now, since it's not required for this change (and may conflict with a priority that a user already set)